test/runner/lib/util.py

```diff
@@ -105,6 +106,7 @@ def raw_command(cmd, capture=False, env=None, data=None, cwd=None, explain=False
     :type explain: bool
     :type stdin: file | None
     :type stdout: file | None
+    :type cmd_verbosity: int | 1
```

Comment:
Only the type is needed here, not the default value, so just `:type cmd_verbosity: int` is needed.